modules/setup/library/Setup/WebWizard.php

```diff
@@ -711,7 +709,23 @@ public function getRequirements($skipModules = false)
                 'setup.requirement.class'
             )
         )));
-        $set->merge($pgsqlSet);
+        $dbSet->merge($pgsqlSet);
+        $set->merge($dbSet);
+
+        $dbRequire = new RequirementSet(true);
```

Comment:
You don't need a requirement set here. Just add the `SetRequirement` directly to `$set`.

Reply:
This is not possible because merge() requires a parameter of type RequirementSet

Reply:
That's why I used `add`, I'm sure there's a corresponding method. :wink:

Reply:
I got it. :)